### ISSUE TITLE
Artwork for Function Finder

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/tasks#789]
Discussed briefly with @clmccutchan a bit already, but summarizing here. We are looking for a few "scenes" that can be interacted with using a spinner. To start, 
- [ ] fish in fish bowl (0 - 4 fish total) 
- [ ] "toon" heart that will shrink a tiny bit
- [ ] snow pile in front of a ruler (ranges from 15 - 18 inches)

Once we have a few started we can add more to the list.

**Comment by clmccutchan** [comment]
@amanda-phet  I should have a draft of all of these for you by Monday. For the fish, would you like one kind of fish or four kinds of fish to put in the fish bowl?

**Comment by amanda-phet** [comment]
Awesome! We can keep it simple and just have one kind of fish.

**Comment by clmccutchan** [comment]
@amanda-phet Here's a first draft of those illustrations. I did draw two kinds of fish, but the second fish is just tweaked from the first fish. I can change the fish color if you want. The fish tank contains some taller plants which I've placed behind the fish. I've also given you two snow piles to choose from: one with very little detail and the other with more detail. Let me know what you would like me to edit!

![789_functionfinder_all](https://cloud.githubusercontent.com/assets/25354670/23875162/0d6fbb7e-07fe-11e7-9f69-2c7bbf9a2d45.png)
